src/Menu/AdminMenuListener.php

```diff
@@ -8,7 +8,7 @@
 use BeHappy\SyliusRightsManagementPlugin\Entity\GroupInterface;
 use BeHappy\SyliusRightsManagementPlugin\Service\GroupServiceInterface;
 use Sylius\Bundle\UiBundle\Menu\Event\MenuBuilderEvent;
-use Symfony\Component\Security\Core\Authentication\Token\Storage\TokenStorage;
+use Symfony\Component\Security\Core\Authentication\Token\Storage\UsageTrackingTokenStorage;
```

Comment:
Why that change ?

Reply:
![image](https://user-images.githubusercontent.com/9592546/105982940-a7222a00-60a0-11eb-90e1-71f142cafd7e.png)

Reply:
in `services.yaml` file you are passing '@security.token_storage'
and when i debug the container in using `debug:container security.token_storage`
i got this output 
```
 ---------------- ---------------------------------------------------------------------------------------- 
  Option           Value                                                                                   
 ---------------- ---------------------------------------------------------------------------------------- 
  Service ID       security.token_storage                                                                  
  Class            Symfony\Component\Security\Core\Authentication\Token\Storage\UsageTrackingTokenStorage  
  Tags             kernel.reset (method: disableUsageTracking)                                             
                   kernel.reset (method: setToken)                                                         
  Public           yes                                                                                     
  Synthetic        no                                                                                      
  Lazy             no                                                                                      
  Shared           yes                                                                                     
  Abstract         no                                                                                      
  Autowired        no                                                                                      
  Autoconfigured   no                                                                                      
 ---------------- ---------------------------------------------------------------------------------------- 
```

Reply:
It should rather be changed to `TokenStorageInterface` then
And also be changed in constructor because you only changed namespace here

Reply:
i tested this too and it is working fine

Reply:
You also need to change the other occurrences in the file